cypress/README.md

```diff
@@ -1,141 +1,50 @@
-4. `@[bug-number]` for example `@FE-1234` - use this tag after `@ignore` to explain why the test is ignored.
-5. `@ignore` and `@[bug-number]` should be removed after the bug fix.
-6. `@accessibility` tests verify accessibility violations.
-7. `@validations` tests verify validation components.
-8. `@themes` tests verify themes colouristics.
-9. `@applitools` visual tests for components based on comparing screenshots base / actual.
+Instructions to integrate Applitools with GitHub can be found [here](https://applitools.com/docs/topics/integrations/github-integration.html).
 
-## Files structure
-```
-.
-├── cypress
-│ ├── fixture
-│ ├── features
-│ │   ├── regression
-│ │       ├── accessibility
-│ │       ├── designSystem
-│ │       ├── experimental
-│ │       ├── test
-│ │       ├── themes
-│ │       ├── validations
-│ │       └── common
-│ │   └── visual
-│ ├── locators
-│ │   └── [component-name]
-│ │       ├── index.js (exported arrow functions for locators)
-│ │       └── locators.js (string const locators)
-│ ├── plugins
-│ └── support
-│     └── step-definitions
-│         └── [component-name]-steps.js (files with cucumber steps)
-├── .eslintrc.json
-├── README.md
-└── tsconfig.json
-```
-
-## Locators
-### How to locate elements
-Always use unique selectors to locate elements with order below:
-1. `[data-component="unique-tag"]` for component and `[data-element="unique-tag"]` for component's element.
-2. `#id`.
-3. `.class-name`.
-
-### How to use locators
-1. Locators are used, the same as step-definitions, per component.
-2. Common locators should be used when the components / elements are common for all stories.
-3. Locators should be named:
-  * Component on preview - `componentPreview`;
-  * Knobs - `componentKnobs`.
+Details of how to write Cypress tests, including coding standards and Applitools integration, can be found in the [Carbon Testing Styleguide](https://github.com/Sage/carbon/blob/master/docs/testing-styleguide.md).
 
 ## Continuous Integration (CI)
-Every commit / pull request in repository initializing cypress tests and runs:
+Every commit/pull request in the repository initiates Cypress test runs. Visual regression tests run in Travis. All other regression tests run in GitHub Actions.
+
 ### Travis
-1. `npm run lint` - runs lint test.
-2. `npm test` - runs jest tests.
-3. `npm start` - runs storybook.
-4. `wait-on http://localhost:9001` - waits until storybook is up and running and is ready to run tests.
-5. `npm run cypress:ci:visual` - run all `visual` test suite.
-### GitHub Actions
-1. `npm start` - runs storybook.
-3. `wait-on http://localhost:9001` - waits until storybook is up and running and is ready to run tests.
-4. `npx cypress run --parallel --record` - run all tests suite.
-  * NOTE: if the tests failed you need to make another `commit`/`push` to generate `cypress run_id` to make possible to re-run cypress tests (on each run `run_id` should be unique).
+1.	`npm start` - runs Storybook.
```

Comment:
I think we can move:
``` 
1.	`npm start` - runs Storybook.
2.	`wait-on http://localhost:9001` - waits until Storybook is up and running and is ready to run tests.
```
this information into section above, before the `Travis`section to unify and not duplicate information here and in `GitHub Actions` section as some `Background` or `Pre-setup`.